src/nvim/ui.c

```diff
@@ -533,3 +533,19 @@ bool ui_is_external(UIExtension widget)
 {
   return ui_ext[widget];
 }
+
+Array ui_list(void)
```

Comment:
renamed to `ui_array` for reason explained in https://github.com/neovim/neovim/pull/8060#discussion_r170536882